modules/hostap/Kconfig

```diff
@@ -8,7 +8,11 @@
 config WIFI_NM_WPA_SUPPLICANT
 	bool "WPA Suplicant from hostap project [EXPERIMENTAL]"
 	select POSIX_CLOCK
+	select POSIX_SIGNAL
+	select POSIX_API
+#	depends on !POSIX_API
```

Comment:
Does this work fine? I was seeing issues with POSIX implementation in Zephyr?

Reply:
Should work now ok when using picolibc